### PR TITLE
🐛 Fix `amp-audio` visual parity on safari browser

### PR DESCRIPTION
**Pull request by deepaklalwani97**
This PR fixes visual parity for amp-component on the safari browser
- The amp-audio looked different on safari browsers compared to the chrome and firefox browsers. Because of width property in `i-amphtml-fill-content` class.  This PR overrides the width property to fix the audio tag UI. The UI for audio tag stays same for the other browsers.
- Added amp-audio component to the storybook for better testing and comparison with all layouts.

### Snapshots
On safari browsers
Before:
Screenshot 2021-10-08 at 12 32 32 PM

After:
Screenshot 2021-10-08 at 12 30 53 PM

Fixes: #36292

### CODE REVIEW

File: extensions/amp-audio/0.1/amp-audio.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+amp-audio audio.i-amphtml-fill-content {
+  width: 100%;
```
Comment (kvchari) [inline extensions/amp-audio/0.1/amp-audio.css:2]: @alanorozco it looks like this works fine, but I'm curious why width is explicitly set to `0` for i-amp-fill-content. According to the comment here https://github.com/ampproject/amphtml/blob/main/css/ampshared.css#L185 it was a workaround for an iOS issue that's been fixed. Can/should that get removed?

Comment (alanorozco) [reply extensions/amp-audio/0.1/amp-audio.css:2]: I'm not entirely sure that this addresses the bug fixed on iOS. On this diff, we mention the WebKit bug, and later include the issue mention for `width: 0`. 

https://github.com/ampproject/amphtml/pull/11134/files

In any case, we should reassess the workaround separately (#36303)

File: build-system/compile/bundles.config.extensions.json [hunk 2]
```diff
@@ -164,7 +164,10 @@
   {
     "name": "amp-audio",
     "version": "0.1",
-    "latestVersion": "0.1"
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true
```
Comment (alanorozco) [inline build-system/compile/bundles.config.extensions.json:169]: Since the CSS required is rather small, I would suggest against adding a separate file.

I would rather set inline styles on the `<audio>` element directly on `buildAudioElement()`, and revert the changes to `hasCss` and removing the `.css` file. You can use the `setStyle` helper.

Please also add a comment to that section:

```js
// TODO(https://go.amp.dev/issue/36303): We explicitly set width 100% to workaround
// an issue where `<audio>` does not fill the parent container on iOS (https://go.amp.dev/issue/36292).
// This is required since global styles for `.i-amphtml-fill-content` set width to 0 in
// order to address a separate bug. Re-assess whether that workaround is needed, and
// remove this style if so.
```

**Review by alanorozco — CHANGES_REQUESTED** [review]
Hi @deepaklalwani97. Thanks for this fix.

The method for the workaround is fine, but I believe we should set inline styles instead. Otherwise it would be somewhat wasteful to create a `<style>` element when we only need one particular property.
